Makefile

```diff
@@ -105,6 +108,9 @@ test-unit: $(SETUP_ENVTEST) ## Run the unit tests
 e2e: KIND_CLUSTER_NAME=operator-controller-e2e
 e2e: run kind-load-test-artifacts test-e2e kind-cluster-cleanup ## Run e2e test suite on local kind cluster
 
+operator-developer-e2e: KIND_CLUSTER_NAME=operator-controller-e2e  ## Run operator-developer e2e on local kind cluster
```

Comment:
Should this have a different `kind` cluster name than the `e2e` test?

Reply:
Have used a different `kind` cluster name : `operator-controller-op-dev-e2e`. Is this a good name?